src/TLuaInterpreter.cpp

```diff
@@ -13493,98 +13493,98 @@ void TLuaInterpreter::initLuaGlobals()
 
     error = luaL_dostring(pGlobalLua, "require \"rex_pcre\"");
     if (error != 0) {
-        string e = "no error message available from Lua";
+        QString e = tr("no error message available from Lua");
         if (lua_isstring(pGlobalLua, -1)) {
-            e = "Lua error:";
+            e = tr("Lua error:");
```

Comment:
A separate issue that does still need to be fixed - there needs to be a space after the colon in English. Otherwise the appended error message is appended directly to it! (BTW for French there will also be a space *before* the colon - but then the French are different that way about punctuation and spaces, « bizarre »!)

Reply:
Noted as Issue: https://github.com/Mudlet/Mudlet/issues/2124 ...